pkg/services/ngalert/notifier/channels/googlechat.go

```diff
@@ -67,6 +71,11 @@ func (gcn *GoogleChatNotifier) Notify(ctx context.Context, as ...*types.Alert) (
 		})
 	}
 
+	if tmplErr != nil {
+		gcn.log.Debug("failed to template Google Chat message", "err", tmplErr.Error())
```

Comment:
I think Debug is too low severity for such errors. I think it should be Warn or at least Info.